apps/backend/src/modules/activity-logs/activity-log.service.ts

```diff
@@ -176,9 +176,14 @@ class ActivityLogService implements Service {
 			items: Object.entries(contributorMap).map(([uniqueKey, commitsArray]) => {
 				const [contributorName, contributorId] = uniqueKey.split("_");
 
+				const contributor = allContributors.items.find(
+					(contributor) => contributor.id === Number(contributorId),
+				);
+
 				return {
 					commitsNumber: commitsArray,
 					contributorId: contributorId ?? "",
+					contributorIsHidden: contributor ? contributor.isHidden : false,
 					contributorName: contributorName ?? "",
```

Comment:
maybe we can return contributor as an object and id, isHidden, and name as properties, or is there an issue? cc @sandrvvu